src/js/tech/html5.js

```diff
@@ -321,7 +321,7 @@ class Html5 extends Tech {
     if ('webkitDisplayingFullscreen' in video) {
       this.one('webkitbeginfullscreen', function() {
         this.one('webkitendfullscreen', function() {
-          this.trigger('fullscreenchange');
+          this.trigger('fullscreenchange', 'not-fullscreen');
```

Comment:
Do you have any specific thoughts/concerns on it? I can't really think of a better way. It doesn't require the tech to know about the player, so that's good. I think we need to make sure this event doesn't bubble, and #2351 isn't finished yet. We might be seeing multiple fullscreen events at the player level right now.